### DISCUSSION TITLE
Games

### DISCUSSION THREAD

**Discussion by FrankHileman** [microsoft/terminal#12631] [Q&A] [answered]
Of course, Windows Terminal can be used for productive tasks occasionally. But where are the games? I really want to see some blocky graphics.

**Comment by j4james** [comment, marked as answer]
I've got a few games I've been working on as a demonstration of what you can do in the terminal, but they're still a work in progress at the moment, and Windows Terminal doesn't yet have all the required functionally.

But as a teaser, here's a short video of one them in action (you may need to unmute the volume if you want to hear the cheesy sound effects).

https://user-images.githubusercontent.com/4181424/161449447-3061e9bb-0cf0-4320-a526-dad69c2d2c5e.mp4

**Reply by FrankHileman** [reply]
I did look at this but I forgot to write a reply. It looks really great and I don't care how blocky the games are... I am nostalgic. I just wish someone was working on this or had finished some. It sounds like a low priority... as if actually typing useful commands should be more important with a terminal... sad.

**Reply by DHowett** [reply]
Hey, so long as people write games... we're happy to make them work right! :smile:
